crates/hive-utils/src/madara/utils.rs

```diff
@@ -402,9 +404,10 @@ mod tests {
         // Create an atomic reference to the test environment to avoid dropping it
         let env = Arc::clone(&test_environment);
         // It is not possible to block the async test task, so we need to spawn a blocking task
+        let handle = tokio::runtime::Handle::current();
```

Comment:
can't the handle be define in the closure?

Reply:
i tried, it doesn't seem so easy